tests/unit/h_vialib/client_test.py

```diff
@@ -0,0 +1,49 @@
+from h_matchers import Any
+
+from h_vialib import ViaClient, ViaDoc
+
+
+class TestViaDoc:
+    def test_no_content_type(self):
+        doc = ViaDoc("http://random.document.com")
+        assert doc.is_pdf == False
+
+    def test_explicit_content_type(self):
+        doc = ViaDoc("http://random.document.com", content_type="pdf")
+        assert doc.is_pdf == True
+
+    def test_url_matches_gdrive_pdf(self):
+        doc = ViaDoc("https://drive.google.com/uc?id=0&export=download")
+        assert doc.is_pdf == True
+
+
+class TestViaClient:
+
+    DEFAULT_OPTIONS = {
```

Comment:
```suggestion
    DEFAULT_OPTIONS = {
```

Also, if these are only used in one test, it might make it more readable to inline them where they are used.